### ISSUE TITLE
Unsupervised Non-intrusive load Monitoring

### ISSUE THREAD

**Issue by Ryton** [opengridcc/opengrid#57]
Could use the labels: "Far_fetched goal" / "Utopia" :-) 

there are quite a few interesting, recent articles/phd-theses  on the topic of (unsupervised) disaggregation of metering data, eg.:
* http://eprints.soton.ac.uk/364263/1.hasCoversheetVersion/Parson-thesis.pdf
* http://web.engr.illinois.edu/~hanj/pdf/sdm11_hkim.pdf

Also, there is a very good framework present in the nilmtk community 
https://github.com/nilmtk/nilmtk , building upon the nomenclature from project haystack & others.

PS: I dont think we should/can go as far as down to the appliance-level classification, but e.g. even splitting water or electricity data, in 2-3-(4?) bins each (vampire load, repetitive loads e.g.freezer/refrigerator, large constant blocks during day (lighting, appliances) and smaller high-peak loads (oven, microwave, etc) could be usefull/doable imho.
Myself, I'm especially interested in water disagregation!

@saroele: This may be interesting to a matematician/programmer wizz with plenty of spare time? ;-)

**Comment by saroele** [comment]
Yes, there are quite a few things going on in nilm.  It looks like nilmtk is a very good starting point, and 100% python. 

When we want to find a mathematician, we should look more into the mathematics and computer science depts of KU Leuven, but I don't have contacts there.  And we need to be able to guide them...  Applying the nilmtk to opengrid should be something we can do ourselves of course.  Stuff for next winter maybe?

**Comment by Ryton** [comment]
<Bump> Winter coming up! ;-)

**Comment by saroele** [comment]
Haha,  great,  this time of long dark nights.   When the only sound is the
crackling of the fire and the tickling of the keyboard of the opengrid
coders...

Now let's hope winter really comes.  Climate change can't take away the
long nights,  but winter is not the same without the cold!
Op 6 nov. 2015 08:44 schreef "J. Ver." <notifications@github.com>:

**Comment by Ryton** [comment]
And another winter in the making!
